packages/web/src/scss/components/FileUploader/README.md

```diff
@@ -329,6 +367,16 @@ const fileList: File[] = myUploaderInstance.getFileQueue();
 const input = myUploaderInstance.inputElement; // Returns an input element, for further use
 ```
 
+### JavaScript Events
+
+| Method                      | Description                                                                                                                                 |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `queueFile.fileUploader`    | This event is fired just before the file was added to the queue.                                                                            |
+| `queuedFile.fileUploader`   | This event is fired after the file was added to queue.                                                                                      |
+| `enqueueFile.fileUploader`  | This event is fired just before the file was removed from the queue.                                                                        |
```

Comment:
doesn't enqueue mean added to the queue? If so, it doesn't make sense? Or maybe I don't understand.

Reply:
Maybe it should be dequeue?

Reply:
Ah I see, it is probably a typo? (also in the second enqueue

Reply:
fixed